view/adminhtml/web/js/new-shipment.js

```diff
@@ -14,14 +14,27 @@ define(
         initialize: function(options) {
           var carriers = document.querySelectorAll('[name="mypa_carrier"]');
           var packageTypes = document.querySelectorAll('[name="mypa_package_type"]');
+          var toggleMyParcel = document.getElementById('mypa_create_from_observer');
           this.mypa_carrier = options.carrier || 'postnl';
           this.mypa_package_type = options.packageType || 'package';
           this.leaveAllVisible = false;
           this.initializeSelectors(carriers);
           this.initializeSelectors(packageTypes);
+          this.initializeToggle(toggleMyParcel, 'mypa-options');
```

Comment:
Als je deze class `js--<naam>` noemt is het voor latere lezers duidelijk dat deze class een functioneel levensdoel heeft en niet voor styling is

Reply:
Goed!